Bottom: 5d95c55f0b1b8d218c042cb0fa262be85043e8f5
Top:    963c3e7b68d4a759f76c58eaa780d3484637a4de
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:18:18 -0500

Refresh of plugin-info

---

diff --git a/plugin.info.txt b/plugin.info.txt
index d9a69bb..d403cff 100644
--- a/plugin.info.txt
+++ b/plugin.info.txt
@@ -1,7 +1,7 @@
 base   tablecalc
 author Christopher Voltz
 email  cjunk@voltz.ws
-date   2016-04-29
+date   2017-05-01
 name   Table Calculations Plugin
 desc   Adds limited spreadsheet functionality to tables
 url    https://github.com/cvoltz/tablecalc.git
